src/sentry/snuba/metrics/fields/base.py

```diff
@@ -235,11 +241,21 @@ class AliasedDerivedMetric(AliasedDerivedMetricDefinition, MetricObject):
     """
 
     def generate_metric_ids(self, projects: Sequence[Project]) -> Set[int]:
-        return {resolve_weak(org_id_from_projects(projects), self.raw_metric_mri)}
+        return {
```

Comment:
same here, this module is generic. the use case needs to be passed down from get_series